README.md

```diff
@@ -159,7 +159,18 @@ If you are using HuaweiCloud IEF, then the edge node you created should be runni
  cd <path to package to be tested>
  go test -v
  
- ```
+ ``` 
+### Run Integration Tests
+
+```shell 
```

Comment:
the content of this section is the same as the one in the following section link. can we remove the next section if not required ? [ Details and use cases section ]